features/step_definitions/factory_steps.rb

```diff
@@ -17,6 +16,35 @@
   end
 end
 
+Given /^I accept a proposed organisation called "(.*?)" with email "(.*?)"$/ do |name, email|
+  steps %{
+    Given the following proposed organisations exist:
+      | name    | description         | address        | postcode | telephone | website            | email    | donation_info        | non_profit |
+      | #{name} | Mourning loved ones | 30 pinner road | HA5 4HZ  | 520800000 | http://#{name}.org | #{email} | www.pleasedonate.com | true       |
+    When I accept the proposed_organisation named "#{name}"
+  }
+end
+
+Then /^"(.*?)" is (notified|invited|ignored) as an organisation admin of "(.*?)"$/ do |email, action, name|
+  case action
+  when 'notified'
+    steps %{
```

Comment:
I've generally tried to avoid nested steps - I've found them really difficult to debug - might we prefer to extract methods from the steps we want to re-use and them re-use those?